compiler/damlc/daml-stdlib-src/DA/Next/Set.daml

```diff
@@ -96,7 +96,11 @@ union (Set t1) (Set t2) = Set $ TextMap.union t1 t2
 intersection : MapKey a => Set a -> Set a -> Set a
 intersection s1 s2 = filter (`member` s2) s1
 
--- | Difference of two sets.
+-- | `difference x y` returns the set consisting of all
+-- elements in `x` that are not in `y`.
+--
+-- >>> fromList [1, 2, 3] `difference` fromList [1]
+-- fromList [2, 3]
```

Comment:
What happens if I do this?

```daml
fromList [1, 2, 3] `difference` fromList [1, 4]
```

Reply:
Updated the example although I hope the text is also clear enough to answer this.